### PR TITLE
feat(specs): update predict client API URL

### PR DESCRIPTION
**Pull request by bengreenbank**
## 🧭 What and Why

🎟 JIRA Ticket: [PRED-560](https://algolia.atlassian.net/browse/PRED-560)
Related PR: https://github.com/algolia/doc/pull/7336

### Changes included:

- Update Predict Client API URL

## 🧪 Test

New URL should be https://predict.us.algolia.com/ or https://predict.eu.algolia.com/ depending on region.

### CODE REVIEW

File: specs/predict/spec.yml [hunk 1]
```diff
@@ -10,17 +10,17 @@ components:
     apiKey:
       $ref: '../common/securitySchemes.yml#/apiKey'
 servers:
-  - url: https://predict-api-432xa6wemq-{region}.a.run.app
+  - url: https://predict-{region}.algolia
```
Comment (shortcuts) [inline specs/predict/spec.yml:13]: I believe it should end with `.com` (see other `spec.yml` files for example)

File: specs/predict/spec.yml [hunk 2]
```diff
@@ -10,17 +10,17 @@ components:
...
     variables:
       region:
         enum:
-          - ue
-          - ew
-        default: ew
+          - us
+          - eu
+        default: eu
```
Comment (shortcuts) [inline specs/predict/spec.yml:19]: The tests should be updated with the region change https://github.com/algolia/api-clients-automation/blob/main/tests/CTS/client/predict/parameters.json#L14

**Review by shortcuts — COMMENTED** [review]
Thanks for the PR :D

File: specs/predict/spec.yml (hunk 2 above)
Comment (bengreenbank) [reply specs/predict/spec.yml:19]: [11f1b82](https://github.com/algolia/api-clients-automation/pull/1010/commits/11f1b824091cf808fa19dcf86a20648c2f8874ed)

**Review by millotp — APPROVED** [review]
Looks good ! Thanks 😄

**Pull request merged**